distribution/docs/src/main/resources/_contents/_metrics/viewing-metrics-tab-contents.adoc

```diff
@@ -0,0 +1,14 @@
+
+==== Viewing Metrics in the ${admin-console}
+
+The Metrics tab in the ${admin-console} (under ${ddf-platform}) has reports in various formats.
+
+. Click on the PNG link for the Catalog Queries row under the column labeled 1d, which stands for one day. A graph of the catalog queries executed in the last 24 hours will be displayed.
```

Comment:
Should point out that I misread `1d` as `id` several times, and that made the rest of the sentence make no sense. Maybe we could: 
* Put `1d` in quotes? 
* Put `one day` in parenthesis instead of comma-delimiting it